Algorithms/Numeric/GaussJordanElimination.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System;
+
+namespace Algorithms.Numeric
+{
+    public static class GaussJordanElimination
```

Comment:
Please make this class not static so that it can be injected into encoder via constructor injection. We will create single interface for linear equation system solvers later and Hill's encoder will be able to use any of them.